flambe/learn/train.py

```diff
@@ -245,20 +261,20 @@ def _aggregate_preds(self, data_iterator: Iterator) -> Tuple[torch.Tensor, torch
 
         Returns
         -------
-        Tuple[torch.tensor, torch.tensor]
-            The predictions and targets.
+        Tuple[torch.tensor, torch.tensor, float]
+            The predictions, targets and mean loss.
 
         """
-        preds, targets = [], []
+        preds, targets, loss = [], [], []
         for batch in data_iterator:
-            batch = self._batch_to_device(batch)
-            pred, target = self.model(*batch)
+            pred, target, batch_loss = self._compute_batch(batch)
+            loss.append(batch_loss.item())
             preds.append(pred.cpu())
             targets.append(target.cpu())
-
+        loss = sum(loss)/len(loss)
```

Comment:
loss.mean()?

Reply:
the losses there are a list of floats. I can change it to be a list of detached loss tensors, but then I'd need to do a concatenation first.

Reply:
Need a space between the operator /. Probably why the test is failing